Application/EdFi.Ods.Api/Security/Authorization/ViewNameHelper.cs

```diff
@@ -18,48 +19,28 @@ public static class ViewNameHelper
         /// Gets the convention-based authorization view name (without the schema), constructed by sorting the specified
         /// endpoint names alphabetically and adding the specified modifier as a suffix (if present).
         /// </summary>
-        /// <param name="endpoint1">The name of one endpoint represented in the view.</param>
-        /// <param name="endpoint2">The name of the other endpoint represented in the view.</param>
+        /// <param name="subject">The name of one endpoint represented in the view.</param>
+        /// <param name="claim">The name of the claim represented in the view.</param>
         /// <param name="authorizationPathModifier">The path modifier for authorization, if applicable, indicating an alternative mechanism for
         /// establishing a relationship between the two supplied endpoints.</param>
         /// <returns>The authorization view name (without the schema).</returns>
-        public static string GetAuthorizationViewName(string endpoint1, string endpoint2, string authorizationPathModifier)
+        public static string GetAuthorizationViewName(string subject, string claim, string authorizationPathModifier)
         {
-            if (endpoint1.EqualsIgnoreCase("StudentUSI") && endpoint2.EqualsIgnoreCase("EducationOrganizationId"))
-            {
-                return $"{endpoint1}To{endpoint2}{authorizationPathModifier}";
-            }
-
-            if (endpoint1.EqualsIgnoreCase("StaffUSI") && endpoint2.EqualsIgnoreCase("EducationOrganizationId"))
-            {
-                return $"{endpoint1}To{endpoint2}{authorizationPathModifier}";
-            }
-
-            if (endpoint1.EqualsIgnoreCase("ParentUSI") && endpoint2.EqualsIgnoreCase("EducationOrganizationId"))
-            {
-                return $"{endpoint1}To{endpoint2}{authorizationPathModifier}";
-            }
-
-            if (string.Compare(endpoint1, endpoint2, StringComparison.InvariantCultureIgnoreCase) < 0)
-            {
-                return $"{endpoint1}To{endpoint2}{authorizationPathModifier}";
-            }
-
-            return $"{endpoint2}To{endpoint1}{authorizationPathModifier}";
+            return $"{subject}To{claim}{authorizationPathModifier}";        
```

Comment:
Just a side note here: This will need to be reversed when we clean up the views (so that it is _claim-to-subject_).

Reply:
Can we reverse the order of the parameters with a refactoring of the signature on this? I'd like to see the claim first, then the subject in the signature.